### ISSUE TITLE
Add Pokedex Entries for Pokemon Formes

### ISSUE THREAD

**Issue by supermii2** [PokeAPI/pokeapi#902]
Some Pokemon with multiple forms have different Pokedex entries for each form.

E.g. Sandshrew Alola has the following in Sun.
"It lives on snowy mountains. Its steel shell is very hard—so much so, it can't roll its body up into a ball. "

Currently, to my knowledge (forgive me if I'm wrong), only pokedex entries for the base form are available, under pokemon-species. It would be nice to be able to find these pokedex entries for different forms too

**Comment by GreatNovaDragon** [comment]
Support

Should also include Names for the Formes. Eternamax Eternatus is like Endynamax Endynalos in german

**Comment by FallenDeity** [comment]
As of now there isn't really a place to put descriptions of other forms as the only place where forms are mentioned is in the `pokemon-species` endpoint which contains descriptions of the original or base form so most likely place to put a description form wise would now be in the `pokemon` endpoint I suppose.

**Comment by supermii2** [comment]
> As of now there isn't really a place to put descriptions of other forms as the only place where forms are mentioned is in the `pokemon-species` endpoint which contains descriptions of the original or base form so most likely place to put a description form wise would now be in the `pokemon` endpoint I suppose.

I agree with this. The only thing I'm not sure about is whether it the existing entries in `pokemon-species` should be removed.

**Comment by FallenDeity** [comment]
Well i think it would be nice to ping one of the maintainer's to ask for their opinions if they give a go ahead flag we can work on a pr shifting the desc to `pokemon` endpoint

**Comment by simonorono** [comment]
I came across this issue while working on a hobby application. I don't know if changing the flavor text entirely to the `pokemon` endpoint would be the desired solution as it's a major breaking change (perhaps v3?).

A temporary ( :crossed_fingers: ) solution might be to create a relationship for the flavor text of pokémon formes and have it only for those forms (anything with id above 10000).

**Comment by FallenDeity** [comment]
solved with #1629 now we have ability to add flavor text for forms data needs to be added though https://pokeapi.co/api/v2/pokemon-form/sandshrew-alola
